ssz-rs/src/bitvector.rs

```diff
@@ -94,7 +93,7 @@ impl<const N: usize> Bitvector<N> {
 
     fn pack_bits(&self) -> Result<Vec<u8>, MerkleizationError> {
         let mut data = vec![];
-        let _ = self.serialize(&mut data)?;
+        let _ = self.serialize(&mut data);
```

Comment:
keep this `?`?

Reply:
Addressed in https://github.com/ralexstokes/ssz-rs/pull/25/commits/438a003498f6a5be024b30a8137c8673eb05895a.